x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -564,11 +568,24 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
           </EuiTitle>
         }
         description={
-          <FormattedMessage
-            id="xpack.snapshotRestore.restoreForm.stepLogistics.includeGlobalStateDescription"
-            defaultMessage="Restores templates that don’t currently exist in the cluster and overrides
-              templates with the same name. Also restores persistent settings."
-          />
+          <>
+            <FormattedMessage
+              id="xpack.snapshotRestore.restoreForm.stepLogistics.includeGlobalStateDescription"
+              defaultMessage="Restores templates that don’t currently exist in the cluster and overrides
+              templates with the same name. Also restores persistent settings and all system indices."
+            />
+
+            {/* Only display callout if include_global_state is enabled and the snapshot was created by ES 7.12+
+             * Note: Once we support features states in the UI, we will also need to add a check here for that
+             * See https://github.com/elastic/kibana/issues/95128 more details
+             */}
+            {includeGlobalState && semverGt(version, '7.12.0') && (
+              <>
+                <EuiSpacer size="xs" />
```

Comment:
This spacing looked a little tight to me in the screenshot. Might be worth checking with Michael?

Reply:
@mdefazio would you mind taking a look? Screenshot in PR description.

Reply:
Just based on looking at the screenshot, here's a rough guideline: (Spacing is tight, but we want to make sure there's _more_ space between the button and the callout).

![image](https://user-images.githubusercontent.com/3756330/112180566-0c425980-8bd2-11eb-82d2-8c58ece442fc.png)

Reply:
I've changed to spacing from `xs` --> `s` above the callout, and from `l` --> `xl` between the callout and the button. Let me know what you think. @mdefazio I'm not sure if you made a mistake when you said to "drop" 1-2 sizes in the screenshot, when in the original comment you said there should be more space between the button and callout. It's possible I misunderstood the screenshot though 😅 . 

<img width="884" alt="Screen Shot 2021-03-23 at 10 04 55 PM" src="https://user-images.githubusercontent.com/5226211/112243591-39b7f300-8c24-11eb-9632-20e4eb199049.png">

Another idea - we could simplify the callout and only have a `title`. Something like this (not committed):
<img width="850" alt="Screen Shot 2021-03-23 at 10 06 28 PM" src="https://user-images.githubusercontent.com/5226211/112243756-7e438e80-8c24-11eb-8d8d-60042e656173.png">

Reply:
My apologies for the poor direction 🤦 .  I was simply _trying_ to say that yes, there should be more space between the text and the callout, and it should be a bit less than the space between the button and the callout.  So I didn't mean to say you had to change the space between the button and the callout (but the new screenshots look fine)

I like the callout with only the title. It's a bit less obtrusive and feels more straightforward.